### DISCUSSION TITLE
Announcement has poor visibility in dark-mode

### DISCUSSION THREAD

**Discussion by aantn** [pradyunsg/furo#338] [Potential Issue] [answered]
The announcement background remains black which makes it blend with the page background.

**Comment by pradyunsg** [comment, marked as answer]
This is intentional; since I don’t want the announcement to be a source of brightness in dark mode.

You can customise this for your documentation using https://pradyunsg.me/furo/customisation/colors/#defining-overrides-for-defaults. There relevant variables are `color-announcement-background` and `color-announcement-text`.

**Comment by aantn** [comment]
Understood, the variables are useful and I fixed it with them.

If anyone is curious, I'm using a purple-ish color which doesn't add too
much brightness. Feel free to copy from our docs at:

https://docs.robusta.dev/master/
